lib/fluent/plugin/prometheus.rb

```diff
@@ -101,7 +102,8 @@ def initialize(element, registry, labels)
         def labels(record, expander, placeholders)
           label = {}
           @base_labels.each do |k, v|
-            label[k] = expander.expand(v, placeholders)
+            accessor_value = v.call(record)
+            label[k] = expander.expand(accessor_value.nil? ? v.keys : accessor_value, placeholders)
```

Comment:
This allows to use both RecordAccessor and RecordTransformer. IMO it's enough to use either one.
I want a value to be expanded by RecordTransfermer only when the RecordAccessor is applied.
Probably the code would be like this?
```
             label[k] = accessor_value.nil? ?  expander.expand(v.keys, placeholders) : accessor_value 
```

Reply:
Yes, that's better. Pushed changes and tested with nested record + original test suite